Mathlib/AlgebraicTopology/SimplicialSet/Basic.lean

```diff
@@ -61,6 +61,12 @@ instance hasColimits : HasColimits SSet := by
 lemma hom_ext {X Y : SSet} {f g : X ⟶ Y} (w : ∀ n, f.app n = g.app n) : f = g :=
   SimplicialObject.hom_ext _ _ w
 
+-- Even though `NatTrans.comp_app` is a `@[simp]` lemma, `simp` is unable to prove this.
+-- Why is this?
+-- Do we want `SSet.comp_app` as a global `@[simp]` lemma?
```

Comment:
Can `simp` prove this if you pass it `SSet.comp_app`?

Reply:
This is `SSet.comp_app`. My question was meant as "do we want this as a simp lemma?".

Reply:
:facepalm: -- sorry for reviewing while my mind was foggy

Reply:
Anyways, `simp` does not realize that `f` and `g` are natural transformations, because that fact is hidden behind one layer of definitions. So I agree that it would be good to make this lemma a simp-lemma.